app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,2 @@
+class LikesController < ApplicationController
```

Comment:
Like controller and comments Controller should be set under the prototypes directory 

you can make that mechanism by typing  this command

```
rails g controller prototypes/comments
```